autopush/endpoint.py

```diff
@@ -639,10 +722,11 @@ def _create_endpoint(self, result=None):
 
     def _return_endpoint(self, endpoint_data, new_uaid, router=None):
         """Called after the endpoint was made and should be returned to the
-        requestor"""
+        requestor
+        """
```

Comment:
nit: no line-break for single-sentence summaries that happen to be too long for a single-line